Rmarkdown/Multiple_Ascending_Dose_PKPD_continuous.Rmd

```diff
@@ -238,170 +245,92 @@ Plot PD marker against concentration. Do you see any relationship? Does response
 ```{r, cache = TRUE, fig.width = 6, fig.height = 4, warning = FALSE, message= FALSE}
 
 data_to_plot <- list()
-data_to_plot[[1]] <- my.data[my.data$CMT==2&my.data$TRTACT!="Placebo"&my.data$PROFDAY==5,
-                             c("TIME","LIDV","ID","TRTACT","CENS")]
+data_to_plot[[1]] <- pk_data[pk_data$TRTACT_low2high!="Placebo"&pk_data$PROFDAY==5,
+                             c("TIME","LIDV","ID","TRTACT_low2high","CENS" )]
 data_to_plot[[1]]$Concentration <- data_to_plot[[1]]$LIDV
 data_to_plot[[1]]$LIDV <- NULL
 
-data_to_plot[[2]] <- my.data[my.data$CMT==3&my.data$PROFDAY==5,
-                             c("TIME","LIDV","ID","TRTACT")]
+data_to_plot[[2]] <- pd_data[pd_data$PROFDAY==5,
+                             c("TIME","LIDV","ID","TRTACT_low2high")]
 data_to_plot[[2]]$Response <- data_to_plot[[2]]$LIDV
 data_to_plot[[2]]$LIDV <- NULL
 
-data_to_plot2 <- merge(data_to_plot[[1]],data_to_plot[[2]],by=c("TIME","ID","TRTACT"))
+data_to_plot2 <- merge(data_to_plot[[1]],data_to_plot[[2]],by=c("TIME","ID","TRTACT_low2high"))
 
-data_to_plot2$TRTACT <- factor(data_to_plot2$TRTACT, levels = rev(levels(data_to_plot2$TRTACT)))
+data_to_plot2$TRTACT_low2high <- factor(data_to_plot2$TRTACT_low2high, levels = rev(levels(data_to_plot2$TRTACT_low2high)))
 
-gg <- ggplot(data = data_to_plot2, aes(x=Concentration,y=Response))+theme_bw()
-gg <- gg + AnnotateStatus(draft.flag)
-gg <- gg + geom_point(aes(color = TRTACT))
+gg <- ggplot(data = data_to_plot2, aes(x=Concentration,y=Response))
+gg <- gg + xgx_annotate_status(status)
+gg <- gg + geom_point(aes(color = TRTACT_low2high))
 gg <- gg + geom_point(data=data_to_plot2[data_to_plot2$CENS==1,], color="red", shape = 8)
-gg <- gg + ylab("Continuous PD Marker (IU/L)") + xlab("Concentration (ng/mL)")
-gg <- gg + scale_x_log10() + annotation_logticks(base = 10, sides = "b", color = rgb(0.5,0.5,0.5))
+gg <- gg + labs(x=pkconc_label , y=pdconc_label)
+gg <- gg + xgx_scale_x_log10() 
 gg <- gg + guides(color = guide_legend(""))
-gg
-
+print(gg)
 ```
 
 ```{r, cache = TRUE, fig.width = 8, fig.height = 3, warning = FALSE, message= FALSE}
 data_to_plot <- list()
```

Comment:
lines 273 through 297 can be deleted and replaced with the below (reusing the gg from previous section):
gg %+% (data = pkpd_data_wide %>% subset(PROFDAY %in% c(1, 3, 5, 7, 9))) + facet_grid(~DAY_label)